watertap/flowsheets/oaro/oaro_multi.py

```diff
@@ -251,7 +251,7 @@ def build(number_of_stages, erd_type=ERDtype.pump_as_turbine):
     m.fs.costing.reverse_osmosis.factor_membrane_replacement.fix(0.15)
     m.fs.costing.reverse_osmosis.membrane_cost.fix(30)
     m.fs.costing.reverse_osmosis.high_pressure_membrane_cost.fix(50)
-    m.fs.costing.high_pressure_pump.cost.fix(53 / 1e5 * 3600)
+    m.fs.costing.high_pressure_pump.unit_cost.fix(53 / 1e5 * 3600)
```

Comment:
@adam-a-a The best argument for updating the `cost` names is to make code more intuitive when flowsheets are updating the values of these variables.

Reply:
I propose `unit_cost` across the board so that is clear that the cost is in $ per some quantity, as opposed to some absolute cost.